src/sveltekit.ts

```diff
@@ -10,11 +10,20 @@ import { registerDevelopmentDefaults } from "./client/config";
 export { Job, EnqueueJobOptions, DefaultJobOptions, QuirrelJobHandler };
 
 registerDevelopmentDefaults({
-  applicationBaseUrl: "localhost:3000",
+  applicationBaseUrl: "localhost:5173",
 });
 
+declare global {
+  interface Response {
+  }
```

Comment:
Not sure how to do declare a global constructor? `Response` is available in global context in SvelteKit apps.